test/TestUtilities/Test.Utility/SimpleTestSetup/SimpleTestSettingsContext.cs

```diff
@@ -137,5 +138,52 @@ public static void RemoveSetting(XDocument doc, string key)
                 item.Remove();
             }
         }
+
+        // Add NetStandard.Library and NetCorePlatforms to the feed and save the file.
+        public void AddNetStandardFeeds()
+        {
+            var reposRoot = GetRepositoryRootDirectory();
+            var netStandardLibraryPackageFeed = GetNetStandardLibraryFeed(reposRoot);
+            var netCorePlatformsPackageFeed = GetNetCorePlatformFeed(reposRoot);
+
+            Assert.True(Directory.Exists(netStandardLibraryPackageFeed));
+            Assert.True(Directory.Exists(netCorePlatformsPackageFeed));
+
+            var section = GetOrAddSection(XML, "packageSources");
+            AddEntry(section, "NetStandardLibrary", netStandardLibraryPackageFeed);
+            AddEntry(section, "NetCorePlatforms", netCorePlatformsPackageFeed);
+
+            Save();
+        }
+
+        private static DirectoryInfo GetRepositoryRootDirectory()
+        {
+            DirectoryInfo currentDir = new DirectoryInfo(Directory.GetCurrentDirectory());
+            ;            while (currentDir != null)
+            {
+                if (File.Exists(Path.Combine(currentDir.FullName, "NuGet.sln")))
+                {
+                    // We have found the repo root.
+                    break;
+                }
+
+                currentDir = currentDir.Parent;
+            }
+
+            return currentDir;
+        }
+
+        private static string GetNetStandardLibraryFeed(DirectoryInfo reposRoot)
+        {
+            var netStandardLibraryPackageFeed = Path.Combine(reposRoot.FullName, "packages", "netstandard.library");
+            return netStandardLibraryPackageFeed;
+        }
+
+        private static string GetNetCorePlatformFeed(DirectoryInfo reposRoot)
+        {
+            var netCorePlatformsPackageFeed = Path.Combine(reposRoot.FullName, "packages", "microsoft.netcore.platforms");
+            return netCorePlatformsPackageFeed;
+        }
```

Comment:
As per my other comment, I don't know if getting package folders is really needed, but if it is, you could have

```csharp
        private static string GetRepoPackageDirectoryName(DirectoryInfo reposRoot, string packageId)
        {
            var netCorePlatformsPackageFeed = Path.Combine(reposRoot.FullName, "packages", packageId);
            return netCorePlatformsPackageFeed;
        }
```

and call it twice from with different parameters, rather than having two methods.

Reply:
Thanks! Changed.